Makefile

```diff
@@ -25,24 +37,37 @@ lint:
 .PHONY: binary
 binary:
 	@echo "WARNING: binary creates a Linux executable. Use cross for macOS or Windows."
+	# TODO(stevvooe): Remove the use of this script. It does nothing. It is
+	# more complicated because the variables need to be converted back from
+	# shell.
 	./scripts/build/binary
 
 .PHONY: cross
 cross:
+	# TODO(stevvooe): Fix the cross build so this isn't a separate target.
+	# Should simply be able to do `make GOOS=windows` to get a cross-compiled
+	# binary.
 	./scripts/build/cross
 
 .PHONY: dynbinary
 dynbinary:
+	# TODO(stevvooe): Change this to target binary but enable cgo, as that is
+	# the only difference.
 	./scripts/build/dynbinary
 
-.PHONY: watch
-watch:
-	./scripts/test/watch
-
 # Check vendor matches vendor.conf
 vendor: vendor.conf
 	vndr 2> /dev/null
-	scripts/validate/check-git-diff vendor
+	if [ "`git status --porcelain -- vendor/ 2>/dev/null`" ]; then \
```

Comment:
You could also consider doing a straight `git diff --exit-code -- vendor` which will output the diff and fail iff there are changes and succeed silently otherwise. It's not quite as concise as the "These files were changed: LIST", but is probably sufficient (and might even be a bit nicer, since it shows you what changed too).

Or you could do `git diff --exit-code --stat -- vendor/` or `git diff --exit-code --name-only` to just output the histogram or file list (still with the fail on changes behaviour). Lacks the "These files were changed header, but that would be handled with perhaps:
```
        vndr 2> /dev/null
        echo "Checking vendoring hasn't changed"
        git diff --exit-code --names-only -- vendor/
        echo "OK"
```
Note that the `echo "OK"` won't be reached if there is a diff, but you don't need folding/escaping.